schemas/context/implementationdetails.schema.json

```diff
@@ -33,7 +33,19 @@
         "xdm:version": {
           "title": "Version",
           "type": "string",
-          "description": "The version identifier of the API, for example, 'h.18'."
+          "description": "The version identifier of the API, e.g h.18."
+        },
+        "xdm:environment": {
```

Comment:
How is this addition different than https://github.com/adobe/xdm/blob/master/schemas/context/environment.schema.json ?

Reply:
`implementationDetails.environment` is where the event was captured, but `environment.type` is where the event occurred.  For example, for an event that originated from a page load on a website, `environment.type=browser`.  If this event was added to the xdm dataset via a server to server call, `implementationDetails.environment=server`.